src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArray = [];
+  let newObject = { ...state };
+
+  for (const key of actions) {
+    switch (key.type) {
```

Comment:
do not forget to add `default` case
![image](https://user-images.githubusercontent.com/22824431/233627444-f84e7379-e019-47c7-b112-0f1f05cb0e3d.png)